src/tracer.js

```diff
@@ -88,13 +94,19 @@ module.exports.initTrace = function initTrace(
 /**
  * Restarts the tracer. Has to be called after a trace has been sent to reset the tracer
  * and start collecting a new trace
+ * @param {object} runner The current of the current trace
```

Comment:
the current of the current of the current of the current trace?

Reply:
lol